src/main/webapp/popup.js

```diff
@@ -40,6 +41,10 @@ window.closeEventPopup = closeEventPopup;
 window.createNewEvent = createNewEvent;
 
 // TODO (remusn@)
-function addEvent(event) {
+export default function addEventToLocalStorage(event) {
+    const eventManager = new EventManager();
+    eventManager.setup();
 
+    eventManager.addEvent(event);
+    console.log(eventManager.getEvents());
```

Comment:
What exactly is this logging? Maybe make it more specific with an explanation

Reply:
It is not meant ot be here, sorry